### PR TITLE
Better formatting for action inputs on GitHub Action README file 

### PR DESCRIPTION
**Pull request by Mauraza**
Signed-off-by: mauraza [gibone@vmware.com](mailto:gibone@vmware.com)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,18 +98,18 @@ The GitHub Action will find the **{PATH}** template variable and will substitute
         }
       }
     }
-  }        
+  }
 }
 ```
-VIB_ENV variable substitution can be a powerful mechanism to make your workloads more flexible and to reuse pipelines. 
+
+VIB_ENV variable substitution can be a powerful mechanism to make your workloads more flexible and to reuse pipelines.
```
Comment (recena) [inline README.md:105]: We could apply format to \`VIB_ENV\`

File: README.md [hunk 2]
```diff
@@ -56,7 +59,7 @@ With that in mind, you can customize your action as follows:
 
 ## Templating your pipelines via environment variables
 
-Pipelines can be templated via environment variables to allow further customization. Any environment variable that your workflow defines with the **VIB_ENV_** prefix will be substituted by the GitHub Action in the pipeline file before being sent to VIB. Furthermore, the GitHub Action will make this substitution independently of whether you are using the VIB_ENV_ prefix in your pipeline or not. 
+Pipelines can be templated via environment variables to allow further customization. Any environment variable that your workflow defines with the **VIB_ENV_** prefix will be substituted by the GitHub Action in the pipeline file before being sent to VIB. Furthermore, the GitHub Action will make this substitution independently of whether you are using the VIB_ENV_ prefix in your pipeline or not.
```
Comment (recena) [inline README.md:62]: Better \`VIB_ENV_\`

File: README.md [hunk 3]
```diff
@@ -35,16 +35,19 @@ jobs:
...
-* **retry-count:** This is the default number of retries to do in case of failure reaching out to VIB. The default value is **3**.
-* **backoff-intervals:** This is the default backoff time used between each retry in case of failure reaching out to VIB. The default value is **[5000, 10000, 15000]**.
-* **only-upload-on-failure:** This parameter sets whether the GitHub Actions should upload artifacts for every task or only for those tasks that have failed. The default value is **true**.
+| Attribute              | Description                                                                                                                                         | Default value        |
+| ---------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------- |
+| config                 | This is the default folder where the action can find the configuration files for the different tasks that will be executed as part of the pipeline. | .vib                 |
```
Comment (recena) [inline README.md:42]: Alphabetical order

File: README.md [hunk 4]
```diff
@@ -4,11 +4,11 @@
 
 This GitHub Action allows to interact with the VMware Image Builder service from VMware, a.k.a. VIB. VIB is a SaaS service that can be used to Package, Verify and Publish software from Independent Software Vendors (ISVs). This software can be packaged in different formats like for example [Carvel Packages](https://carvel.dev) or [Helm Charts](https://helm.sh).
 
-One of the strongest capabilities of VIB is being able to verify simultaneously your software in multiple Kubernetes distributions and flavours. Currently VIB does support verification in TKG (different versions), GKE, AKS, EKS and IKS with plans to support even more K8s distributions. 
+One of the strongest capabilities of VIB is being able to verify simultaneously your software in multiple Kubernetes distributions and flavours. Currently,d VIB does support verification in TKG (different versions), GKE, AKS, EKS and IKS with plans to support even more K8s distributions.
```
Comment (recena) [inline README.md:7]: Typo _Currently,d VIB does_

File: README.md (hunk 3 above)
Comment (Mauraza) [reply README.md:42]: Updated

**Review by recena — APPROVED** [review]

**Pull request merged**